### ISSUE TITLE
Broken text in menu button

### ISSUE THREAD

**Issue by bkauler** [joewing/jwm#534]
Hi, I recently updated jwm from 1685 to 2.4.0, and have it in the latest  release of EasyOS. A user has reported something, that I should have noticed...

The menu button, label "Menu" at bottom-left of screen, is now showing as "M...u"

		<TrayButton label="Menu" icon="mini-dog.xpm">root:3</TrayButton>

This has been in the "pups" (Puppy Linux) since forever.

Let's see if I can drop a photo:
![jwm-menu-button-text](https://user-images.githubusercontent.com/4200448/144706680-7d7e48fd-4526-42d2-9e10-ffe49b0cff38.png)

**Comment by joewing** [comment]
Thanks for reporting. I think I have a fix (just some incorrect constants used when rendering the text). I'll try to get a new release out soon.

**Comment by maurosh** [comment]
Same issue on archlinux, after upgrading to JWM 2.4.0.
I also faced an issue with svg icons in menus and menubar: some icons appeared in a tiny size and some doesn't appeared at all. In tray area they seems to be fine.
Using a bash script i wrote for updating the menu applications (https://github.com/maurosh/SparkyBonsai/blob/master/menumaker) doesn't worked, exported xml in a flat file doesn't worked as expected too.
Tried to downgrade librsvg (the lib responsible for svg appearance) but it didn't worked. Jwm session refused to start, so i brought back the updated one.
Then i downgraded JWM from cache (v. 2.3.7-3) and my desktop went back to normal.
Waiting for the next release to check.
